src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php

```diff
@@ -29,19 +29,27 @@
 use Sulu\Component\Rest\ListBuilder\Expression\Doctrine\DoctrineWhereExpression;
 use Sulu\Component\Rest\ListBuilder\Expression\Exception\InvalidExpressionArgumentException;
 use Sulu\Component\Rest\ListBuilder\FieldDescriptorInterface;
+use Sulu\Component\Security\Authorization\AccessControl\SecuredEntityRepositoryTrait;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 
 /**
  * The listbuilder implementation for doctrine.
  */
 class DoctrineListBuilder extends AbstractListBuilder
 {
+    use SecuredEntityRepositoryTrait;
```

Comment:
I am using this trait here, because I'd have to copy the entire functionality over here otherwise. That means we would have to change the logic in two places, in case we are changing something here. At first I was not glad about this because the trait has `Repository` in its name, but when I thought about it, the list builder could also be considered as some kind of repository, since it is also loading data.